src/components/NavBar.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState, useContext } from "react";
+import { Route, Routes, Link } from "react-router-dom";
+import axios from "axios";
+import { getAllArticles } from "../utils";
+
+const NavBar = ({}) => {
+  return (
+    <>
+      <Link to="/">
+        <h1>NC NEWS</h1>
+      </Link>
+      <section className="navBar">
+        <button>Home</button>
```

Comment:
While this goes nowhere, buttons should not be used for navigation. Buttons are a functional element that should 'do a thing'. You can use a link and style it to look like a button for sure.